src/smt/expand_definitions.cpp

```diff
@@ -43,6 +43,17 @@ Node ExpandDefs::expandDefinitions(
     std::unordered_map<Node, Node, NodeHashFunction>& cache,
     bool expandOnly)
 {
+  TrustNode trn = expandDefinitions(n, cache, expandOnly, nullptr);
+  return trn.isNull() ? Node(n) : trn.getNode();
+}
+
+TrustNode ExpandDefs::expandDefinitions(
+    TNode n,
+    std::unordered_map<Node, Node, NodeHashFunction>& cache,
+    bool expandOnly,
+    TConvProofGenerator* tpg)
+{
+  Node orig = n;
```

Comment:
```suggestion
  const TNode orig = n;
```
orig is never written to, and `n` is a `TNode` already as well.